capstone-app/src/app/modules/maps/map/map.component.spec.ts

```diff
@@ -1,3 +1,4 @@
+import { MatMenuHarness } from '@angular/material/menu/testing';
```

Comment:
can u move this one line down so the /// <reference  types="googlemaps" /> is the first line.

Reply:
Yes!